RELEASE-NOTES.md

```diff
@@ -3,6 +3,7 @@
 ## PyMC3 3.9 (On deck)
 
 ### New features
+- `pm.sample` now takes 1000 draws and 1000 tuning samples by default, instead of 500 previously (see [#3855](https://github.com/pymc-devs/pymc3/pull/3855)).
```

Comment:
Can you move this under `Maintenance`?

Reply:
Ha ha, I hesitated between the two... and of course chose the wrong option 😅 Will do in a minute

Reply:
Done